install/installer/pkg/config/v1/config.md

```diff
@@ -60,9 +60,12 @@ Config defines the v1 version structure of the gitpod config file
 |`workspace.templates.prebuild`||N|  ||
 |`workspace.templates.imagebuild`||N|  ||
 |`workspace.templates.regular`||N|  ||
-|`workspace.pvc.size`||Y|  |  Size is a size of persistent volume claim to use|
-|`workspace.pvc.storageClass`|string|N|  |  StorageClass is a storage class of persistent volume claim to use|
-|`workspace.pvc.snapshotClass`|string|N|  |  SnapshotClass is a snapshot class name that is used to create volume snapshot|
+|`workspace.prebuildPVC.size`||Y|  |  Size is a size of prebuild workspace persistent volume claim to use|
```

Comment:
Hmm. So, I was thinking if there was a better way to surface this config considering if there would be more things specific to prebuilds here. Without changing existing structure, How about `workspace.prebuild.PVC.*` ? 🤔

Reply:
@Pothulapati Thanks for your feedback.
If we choose the approach `workspace.prebuild.PVC.*`, we need to change the `workspace.pvc.*` to `workspace.regular.pvc.*`.

I think we could consider changing the structure once we want to support more custom settings between prebuild and regular workspace. WDYT?